### ISSUE TITLE
PosixPath object is not subscriptable

### ISSUE THREAD

**Issue by sebsto** [recitale/recitale#25]
Hello Quentin, thank you for maintaining this project.

When trying the latest docker image ( v2.0.0a5) on macOS,  with a very simple directory layout, I received the following error : 
```
$ docker run --rm -v ./working:/var/www ghcr.io/recitale/recitale autogen --all          

WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
2023-08-23 01:32:59,276 WARNING (recitale.py:189) - warning: If you want the rss, OpenGraph and/or the social network share to work, you need to specify the website url in root settings
Traceback (most recent call last):
  File "/usr/local/bin/recitale", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/local/lib/python3.11/site-packages/recitale/recitale.py", line 871, in main
    autogen(args.folder, args.force)
  File "/usr/local/lib/python3.11/site-packages/recitale/autogen.py", line 93, in autogen
    build_template(folder, force)
  File "/usr/local/lib/python3.11/site-packages/recitale/autogen.py", line 72, in build_template
    files_grabbed.extend(glob(Path(".").joinpath(folder, files)))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/glob.py", line 28, in glob
    return list(iglob(pathname, root_dir=root_dir, dir_fd=dir_fd, recursive=recursive,
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/glob.py", line 48, in iglob
    root_dir = pathname[:0]
               ~~~~~~~~^^^^
TypeError: 'PosixPath' object is not subscriptable
```

Here is my directory layout 
<img width="210" alt="image" src="https://github.com/recitale/recitale/assets/401798/a43ef11f-f0a9-4188-bd6f-b18b39531fd9">

What did I do wrong ?

**Comment by sebsto** [comment]
Note : when I manually populate the `sections` part of `settings.yaml` in the folder Colombia, the `build1 command works as expected.

**Comment by QSchulz** [comment]
Thanks for the report. This is actually a duplicate of #16. Fixed in b1511ac2cea25fa0e10093c5cc6bbfdd7059670c, not yet in any release though. I'm pondering whether I want to squeeze a few more things in the release or make one soon. Time will tell.

**Comment by sebsto** [comment]
Thank you ! I might try the current version from git but I am blocked by https://github.com/recitale/recitale/issues/26 :-( Any idea ?  I looked a bit a the cache code and I don't understand why the original developer choose to launch a separate process for this. Any idea ?
